test/integration/framework/perf_utils.go

```diff
@@ -33,29 +32,44 @@ const (
 	retries = 5
 )
 
+// NodeTemplate is responsible for creating a v1.Node instance that is ready
+// to be sent to the API server.
+type NodeTemplate interface {
+	// GetNodeTemplate returns a node template for one out of many different nodes.
+	// Nodes with numbers in the range [index, index+count-1] will be created
```

Comment:
This is a bit confusing. It's not from this range, it's just the number `index`, but the node belongs to a group of `count` nodes.

Reply:
Right, changed